docs/source/install.rst

```diff
@@ -34,3 +38,22 @@ To install dask from source, clone the repository from `github
     git clone https://github.com/blaze/dask.git
     cd dask
     python setup.py install
+
+
+Test
+----
+
+Test dask with ``py.test``::
+
+    cd dask
+    py.test dask
+
+Although please aware that installing dask naively may not install all
+requirements by default.  Please read the ``pip`` section above that discusses
+requirements.  You may choose to install the ``dask[complete]`` which includes
+all dependencies for all collections.  Alternatively you may choose to test
+only certain submodules depending on the libraries within your environment.
+For example to test only dask core and dask array we would run tests as
+follows::
+
```

Comment:
OK with that, but as a packager, I need to install everything with `python setup.py install`. So, I cannot rely on pip to install the dependence. Here, we need to list the modules needed to run the tests (numexpr, partd, ...)